pkgs/development/python-modules/django-rest-swagger/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, coreapi
+, openapi-codec
+, simplejson
+, django
+, djangorestframework
+}:
+
+buildPythonPackage rec {
+  pname = "django-rest-swagger";
+  version = "2.2.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "marcgibbons";
+    repo = pname;
+    rev = version;
+    hash = "sha256-G1cdk4pClHunrr6+4QDvSzx0UrBWa2t97E24PLCrm4s=";
+  };
+
+  propagatedBuildInputs = [
+    coreapi
+    openapi-codec
+    simplejson
+    django
+    djangorestframework
+  ];
+
+  # Test raises django.core.exceptions.ImproperlyConfigured
```

Comment:
Is it possible to run tests with something like
```
  nativeCheckInputs = [
    pytest-django
    pytestCheckHook
  ];

  env.DJANGO_SETTINGS_MODULE = "django-rest-swagger.tests";
```

Reply:
Fixed.